src/utilities/findBreakingChanges.js

```diff
@@ -102,7 +102,11 @@ export function findDangerousChanges(
   return ((dangerousChanges: any): Array<DangerousChange>);
 }
 
-function findSchemaChanges(
+/**
+ * Given two schemas, returns an Array containing descriptions of all the types
+ * of potentially breaking change and dangerous change
+ */
+export function findSchemaChanges(
```

Comment:
@wtrocki Totally reasonable 👍 
Moreover, I think it's confusing to have multiple functions that basically do the same so 
I think `findSchemaChanges` should be a function to rule them all.

Since we working on new major release `15.0.0` I think its good time to remove `findDangerousChanges` and `findBreakingChanges`. And merge `BreakingChange | DangerousChange` into single `SchemaChange` type.

Can you please do that?

Also, don't forget to update TS typings in `tstypes` folder.
Another thing all public APIs should be exported through `src/index.js` and `src/utilities/index.js`.
https://github.com/graphql/graphql-js/blob/master/src/README.md